### ISSUE TITLE
Run unit tests under Windows on Travis

### ISSUE THREAD

**Issue by glyn** [projectriff/riff#1001]
These will fail until https://github.com/projectriff/riff/issues/983 is fixed.

needs https://github.com/projectriff/riff/issues/983

**Comment by glyn** [comment]
This is a dup of https://github.com/projectriff/fats/issues/46.

**Comment by glyn** [comment]
Re-opening as this is a subset of what's required for https://github.com/projectriff/fats/issues/46.

**Comment by georgeharley** [comment]
Travis builds on Windows currently seem to have a problem with calling the "standard" Windows command `rmdir` from a Makefile. A simple target like this, which works fine when the `make` is carried out on a real Windows system...
```
clean-docs:
	rmdir /Q /S docs
```
repeatedly failed in Travis builds. The error was as follows, which looks like some other UNIX-y `rmdir` is somehow getting called...
```
rmdir /Q /S docs
The command "make build test verify-docs" exited with 2.

Done. Your build exited with 1.
rmdir: failed to remove '/Q': No such file or directory
rmdir: failed to remove '/S': No such file or directory
rmdir: failed to remove 'docs': Directory not empty
make: *** [Makefile:85: clean-docs] Error 1
```
Have [asked about the matter](https://travis-ci.community/t/calling-rmdir-from-makefile-not-working/1403) on the Travis community forum. Have tried a couple of other variations on the basic `rmdir` call that also work on a real Windows laptop but stubbornly fail in Travis.
  
While the cross platform Makefile developed for this issue works on real Windows but not on Travis I've configured the new Travis build job to only run the `build` and a `test` targets.

**Comment by fbiville** [comment]
Merged in 8255419f34326980a400d5e944ffe684f733da7f, I believe.
@georgeharley @glyn can this issue be closed?

**Comment by glyn** [comment]
Seems so. Thanks.
